main/src/ca/uwaterloo/flix/language/ast/SyntaxTree.scala

```diff
@@ -280,6 +278,8 @@ object SyntaxTree {
 
       case object LiteralVector extends Expr
 
+      case object LocalDef extends Expr
```

Comment:
Why is this here now? 😕

Reply:
What I mean is: I thought this was _already here_

Reply:
No I just reused the letrec parsing and then renamed it (in this pr)... this is where it should be alphabetically

Reply:
OK